openlibrary/core/sponsorships.py

```diff
@@ -61,6 +62,8 @@ def get_sponsored_editions(user, page: int = 1) -> list:
     :param user user: infogami user
     :return: list of archive.org editions sponsored by user
     """
+    if config.get('offline_mode', False):
+        return []
```

Comment:
Returns empty list of sponsorship data while in offline-mode.  This data is used to populate the sidebar in "My Books" views, even for patrons who did not participate in the program while it was active.